### DISCUSSION TITLE
reconnection issues of paired devices like airpodes and keyboard

### DISCUSSION THREAD

**Discussion by Kushalsharma0702** [blueman-project/blueman#2527] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [x] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

**Comment by Lextor1975** [comment]
The same problem for me. If the PC is not used for a while and the screen goes off, I can only wake up the PC with a wired keyboard. The Bluetooth mouse does not connect. You have to restart Mint 22 or delete the mouse in the Bluetooth devices and reconnect. Is there a workaround for this?
